bin/wasabi.sh

```diff
@@ -42,6 +43,7 @@ options:
   -p | --profile [ profile ]             : profile; default ${profile_default}
   -e | --endpoint [ host:port ]          : api endpoint; default: ${endpoint_default}
   -v | --verify [ true | false ]         : verify installation configuration; default: ${verify_default}
+  -v | --buildtests [ true | false ]     : perform tests after build; default: ${buildtests_default}
```

Comment:
@longdogz you should have a different shorthand from "-v" for this option otherwise it conflicts with the shorthand for --verify. Possibly "-t" ?